bot/exts/evergreen/error_handler.py

```diff
@@ -113,13 +113,23 @@ async def on_command_error(self, ctx: commands.Context, error: commands.CommandE
             return
 
         if isinstance(error, commands.CheckFailure):
-            await ctx.send(embed=self.error_embed("You are not authorized to use this command.", NEGATIVE_REPLIES))
```

Comment:
There wasn't a need to change this line as it is in one line and doesn't fall over to the next line. Basically what I meant was, instead of:
```py
await ctx.send(embed=self.error_embed("This is a long message",
               argument_2)
```
And it should rather be:
```py
await ctx.send(
    embed=self.error_embed("This is a long message",
    argument_2
)
```